common/src/api/internal/shared.rs

```diff
@@ -68,9 +68,17 @@ pub struct SourceNatConfig {
     pub last_port: u16,
 }
 
+// We alias [`RackNetworkConfig`] to the current version of the protocol, so
+// that we can convert between versions as necessary.
+pub type RackNetworkConfig = RackNetworkConfigV1;
+
 /// Initial network configuration
+///
+/// TODO(AJS): It's unclear if this should be serde renamed to `RackNetworkConfig`
+/// I *think* it's useful to have the version be explicit in the name, but I'm open
+/// to discussion.
```

Comment:
I'm not sure serde cares about struct names, does it? At least I think that's true for JSON; if this is a top-level object, only the field names show up, and if this is embedded in another struct or enum, it's named based on that container's field/variant.

Maybe schemars would look at `serde(rename)` rules to pick a name in the OpenAPI spec though? I don't have strong feelings either way on what it's named there.

Reply:
I added this comment because this rewrote the schema. I still think I prefer V1 in the schema explicitly though. WDYT?

Reply:
Leaving the V1 in the schema explicitly seems fine to me :+1: